src/libos/crates/async-rt/src/sync/rwlock/inner.rs

```diff
@@ -0,0 +1,476 @@
+// Copyright (C) 2020 Ant Financial Services Group. All rights reserved.
+
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions
+// are met:
+
```

Comment:
I think the whole copyright block should not be broken by empty line. Break it by empty comment line instead.

Reply:
Done.